exp-models/addon/mixins/jam-document-adapter.js

```diff
@@ -3,7 +3,7 @@ import Ember from 'ember';
 export default Ember.Mixin.create({
     findRecordUrlTemplate: '{+host}/{+namespace}/documents{/id}',
     findAllUrlTemplate: '{+host}/{+namespace}/collections{/jamNamespace}.{+collectionId}/documents',
-    queryUrlTemplate: '{+host}/{+namespace}/collections{/jamNamespace}.{+collectionId}/_search',
+    queryUrlTemplate: '{+host}/{+namespace}/collections{/jamNamespace}.{+collectionId}/documents',
```

Comment:
The same base query URL should be used across all documents (Lauren's original fix was for only experiments).

Ostriches proposed an elegant compromise which takes advantage of the similar parts of the URL, and will allow us to do both searching and querying by making this URL dynamic. I think @leb2dg  is working on that change in a followup PR.